### DISCUSSION TITLE
Parallelization of seqr_loading in local instance

### DISCUSSION THREAD

**Discussion by bnouyou** [broadinstitute/seqr#2283] [Q&A]
Hello,

I have a local instance of seqr. For now, I have loading VCF files one by one.
I would like to know if it's possible to run multiple instance of the pipeline-runner (`seqr_loading SeqrMTToESTask`) at the same time, for loading several samples at the same time ?
Is there a possibility to parrallelize the `seqr_loading SeqrMTToESTask` command ?

Thank you in advance for you help.

**Comment by hanars** [comment]
The `SeqrMTToESTask` task takes in one joint called VCF as the input. We joint call all of our samples together and run this step once for all projects, and then do the export step for individual projects. But If you have your data in multiple VCFs then you need to run it multiple times. In terms of running in parallel, sure you can have multiple processes running at the same time as long as you have sufficient compute resources available
